core/block_test.go

```diff
@@ -238,12 +240,85 @@ func TestBlockHash(t *testing.T) {
 			"transaction hash (%v) at index: %v does not match receipt's hash (%v)",
 			mainnetBlock1.Transactions[1].Hash().String(), 1,
 			mainnetBlock1.Receipts[1].TransactionHash)
-		commitments, err := core.VerifyBlockHash(mainnetBlock1, &utils.Mainnet)
+		commitments, err := core.VerifyBlockHash(mainnetBlock1, &utils.Mainnet, nil)
 		assert.EqualError(t, err, expectedErr)
 		assert.Nil(t, commitments)
 	})
 }
 
```

Comment:
Aneeques PR aims to remove these, so I'll neglect reviewing these specific changes